packages/core/parcel/src/cli.js

```diff
@@ -75,6 +75,11 @@ const commonOptions = {
   '--cache-dir <path>': 'set the cache directory. defaults to ".parcel-cache"',
   '--watch-dir <path>':
     'set the root watch directory. defaults to nearest lockfile or source control dir.',
+  '--watcher-ignore-dirs [path]': `list of directories watcher should not be tracking for changes. defaults to ['.git', '.hg']`,
```

Comment:
I think they don't need to be directories, the watcher also accepts globs. so maybe just `--watcher-ignore`? or maybe `--watch-ignore` for consistency with `--watch-dir`?